components/jwt-sidecar/get-vault-secrets.sh

```diff
@@ -44,6 +45,10 @@ login()
   # fi
   #
   local RT=$?
+  if [[ "$RESP" == *"$TIMEOUT"* ]]; then
+    echo "$TIMEOUT"
+    return
```

Comment:
Actually, it does not matter what value we return from the `login` function, because the calling function, `run` , is using it as follow:
```RESP=$(login "${ROLE}")
```
so it only inspects the result of the `echo` command. The return in login is there only to terminate execution further in the function.